src/main/java/com/mailchimp/MailChimpFactory.java

```diff
@@ -12,13 +13,23 @@ static String mailChimpApi (String i)
         return "https://"+i+".api.mailchimp.com";
     }
 
-    public static MailChimpClient create (String apiKey, String apiBase)
+    public static MailChimpClient createWithBasicAuth (String apiKey, String apiBase)
     {
         MailChimpClient mailChimp = Feign.builder()
```

Comment:
No problem. I am also adding additional functionality like creating lists and batches. I will make separate pull requests for them.